village/src/components/Smurfs.js

```diff
@@ -1,26 +1,52 @@
 import React, { Component } from 'react';
 
 import Smurf from './Smurf';
+import styled from 'styled-components';
+
+
+const SmurfCards=styled.div`
+
+background:lightblue;
+color:black;
+border:1px solid pink;
+width:200px;
+margin:auto;
+margin-top:5px;
+display:flex;
+justify-content:center;
+`
+const SmurfPage=styled.div`
+color:white;
+width:800px;
+margin:auto;
+display:flex;
+justify-content:center;
+flex-flow:wrap;
+`
+
 
 class Smurfs extends Component {
   render() {
     return (
-      <div className="Smurfs">
-        <h1>Smurf Village</h1>
+     <SmurfPage>
+        
         <ul>
           {this.props.smurfs.map(smurf => {
             return (
-              <Smurf
+              <SmurfCards><Smurf
```

Comment:
`SmurfCards` seems to be the styling for the `Smurf` component, I would put the styles in Smurf.js